pkgs/top-level/all-packages.nix

```
@@ -34952,6 +34952,8 @@ with pkgs;
 
   lilypond = callPackage ../misc/lilypond { guile = guile_1_8; };
 
+  lilypond-unstable = callPackage ../misc/lilypond/unstable.nix { };
```

Do we want to be specific to avoid issues in the future if `guile` points to e.g. `guile_3_0`?
```suggestion
  lilypond-unstable = callPackage ../misc/lilypond/unstable.nix {
    guile = guile_2_2;
  };
```

---

Looks like no: https://github.com/NixOS/nixpkgs/pull/182614#discussion_r933495425